docs/configuration/metrics/redis-cache.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: default
+title: Azure Cache for Redis Declaration
+---
+
+## Azure Cache for Redis - ![Availability Badge](https://img.shields.io/badge/Available%20Starting-v0.1.0-green.svg)
+You can declare to scrape an Azure Cache for Redis via the `RedisCache` resource type.
+
+The following fields need to be provided:
+- `cacheName` - The name of the Redis Cache service
+
+The list of supported metrics is available on the Azure Cache for Redis [monitoring documentation page](https://docs.microsoft.com/en-us/azure/azure-cache-for-redis/cache-how-to-monitor#available-metrics-and-reporting-intervals).
```

Comment:
Maybe good to point to official overview for Monitor to get all metrics, with a pointer to Redis docs for more info?
```suggestion
All supported metrics are documented in the official [Azure Monitor documentation](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/metrics-supported#microsoftcacheredis).

You can find more documentation on each metric in the Azure Cache for Redis [monitoring documentation](https://docs.microsoft.com/en-us/azure/azure-cache-for-redis/cache-how-to-monitor#available-metrics-and-reporting-intervals).
```

Reply:
yes, totally makes sense.